src/index.js

```diff
@@ -0,0 +1,26 @@
+import { combineReducers } from 'redux';
+
+import createApp from './createApp';
+import createComponent from './createComponent';
+import createFactory from './createFactory';
+import createModel from './createModel';
+import createService from './createService';
+import mapToProps from './components/mapToProps';
+import Model from './Model';
+import PropTypes from './components/mapToProps';
+import Region from './components/Region';
+import render from './render';
+
+export default {
+  combineReducers,
+  createApp,
+  createComponent,
+  createFactory,
+  createModel,
+  createService,
+  mapToProps,
+  Model,
+  PropTypes,
+  Region,
+  render,
```

Comment:
dangle commas help a lot when visualising changes and make it more convenient and less error-prone for manual merges.